pkgs/development/python-modules/python-ndn/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pythonOlder
+, setuptools
+, pycryptodomex
+, lark
+, pygtrie
+, aenum
+, pytestCheckHook
```

`lib` first and the rest of the inputs alphabetically ordered will make the diffs simpler to read in the future.

```suggestion
, aenum
, buildPythonPackage
, fetchFromGitHub
, lark
, pycryptodomex
, pygtrie
, pytestCheckHook
, python3Packages
, pythonOlder
, setuptools
```